### PR TITLE
Adds 5 new dishes made with the more recent material plants.

### PR DESCRIPTION
**Pull request by Eneocho**
Diamond Carrot, Silicate Citrus and Plasma Cabbages had no recipes whatsoever until now:
### Silicate Soup:
image
Surprisingly flavourful considering it's basically sand.
Made with one Silicate Citrus and 10u Water.

### Glass Burger:
image
Makes you scream in pain. You did eat glass after all.
Made with 5u Flour, one Silicate Citrus and a regular glass pane.

### Diamond Fries:
image
Made by processing a Diamond Carrot in the food processor.

### Plasma Stew:
image
Does not contain any plasma and is quite nutritious.
Made with one Meat slab, one Tomato and one Plasma Cabbage.

### Bling Pizza:
spark bling resized spark bling resized slice

Made with 10u Novaflour, one Diamond Carrot, one Gold Apple, one Silver Pear and one Ambrosia Deus.
Centcomm Selects for high rank executives. Contains traces of diamond and is quite spicy.

## What this does
Adds 5 new dishes.
## Why it's good
More recipes is always good. And gives you a reason to make Plasma Cabbages/Silicate Citrus/Diamond Carrots beyond just materials farming.
Names open to change if someone can suggest better ones.

[sprites][content]
:cl:
 * rscadd: Plasma Stew, Silicate Soup, Glass Burger, Diamond Fries and Bling Pizza.
 * imageadd: Icons for the aforementioned dishes.

### CODE REVIEW

**Comment by jwhitak** [comment]
The last one should be added to cargo orders. It will never be filled. But it belongs there.

**Comment by Eneocho** [comment]
> The last one should be added to cargo orders. It will never be filled. But it belongs there.

Good idea. Implemented.

**Comment by Blithering** [comment]
I recommend stealing the sparkle animation from the magnificent magboots and putting them on the bling pizza, might make it look bling-ier

**Comment by Eneocho** [comment]
> I recommend stealing the sparkle animation from the magnificent magboots and putting them on the bling pizza, might make it look bling-ier

I'll see what I can do.

**Comment by Eneocho** [comment]
> > I recommend stealing the sparkle animation from the magnificent magboots and putting them on the bling pizza, might make it look bling-ier
> 
> I'll see what I can do.

Here's what I have:
spark bling resized spark bling resized slice

**Comment by Optimism333** [comment]
Fun additions

File: code/game/centcomm_orders/orders_service.dm [hunk 1]
```diff
@@ -67,6 +67,12 @@
 			)
 			worth = 700*requested[requested[1]]
 			sauce = 2
+		if (8)
+			requested = list(
+				/obj/item/weapon/reagent_containers/food/snacks/sliceable/pizza/blingpizza = rand(2,5)
+			)
+			name = "Central Command's Executive Suite" 
+			worth = 800*requested[requested[1]]
```
Comment (Kurfursten) [inline code/game/centcomm_orders/orders_service.dm]: big reward for something you can get by ordering pizza crates for 75...

Comment (adacovsk) [reply code/game/centcomm_orders/orders_service.dm]: @Eneocho gotta reach the top monthly richest escapee somehow

Comment (Eneocho) [reply code/game/centcomm_orders/orders_service.dm]: I want them available on the pizza cratees, so I'll just remove the request, to be honest I didn't even think about that.

Comment (Eneocho) [reply code/game/centcomm_orders/orders_service.dm]: There, removed.

**Comment by Eneocho** [comment]
Fixed, @Kurfursten.

**Comment by pedr69** [comment]
the blingpizzas should not have a 5 times 20% chance to roll in the normal cargo pizza pack order, in my opinion

**Comment by Blithering** [comment]
> the blingpizzas should not have a 5 times 20% chance to roll in the normal cargo pizza pack order, in my opinion

That is sensible, keep the fun difficult stuff chef (and slime reaction, damn slimes taking our jobs) only.

**Pull request merged**